### PR TITLE
Print dist

### PR DESCRIPTION
**Pull request by lillyfwang**
Opens each plot in different windows at the end of running everything.

### CODE REVIEW

**Comment by lillyfwang** [comment]
**X^2**
screen shot 2015-12-18 at 12 06 09 am
**Uniform**
screen shot 2015-12-18 at 12 06 11 am
**Normal**
screen shot 2015-12-18 at 12 06 16 am

File: compiler/lib/core.py [hunk 1]
```diff
@@ -20,15 +21,29 @@
 DIST_LENGTH = 10000
 SAMPLE_STEP = 100
 
+PLOT = False
+
 def exception(s):
     sys.stderr.write("%s\n" % s)
     exit(1)
 
 def print(*args, **kwargs):
```
Comment (alexandramedway) [inline compiler/lib/core.py]: why is this red on github?

File: compiler/odds.ml [hunk 2]
```diff
@@ -34,7 +34,7 @@ let _ =
           let output_file = Sys.argv.(2) and stdlib_file = Sys.argv.(3) in
           let stdlib = Utils.str_of_file stdlib_file in
           let file = open_out output_file
-          in fprintf file "%s\n\n%s\n" stdlib prog; close_out file
+          in fprintf file "%s\n\n%s\n%s\n" stdlib prog "if PLOT:\n\tplt.show()"; close_out file
```
Comment (alexandramedway) [inline compiler/odds.ml]: Can we move this to another file? And call a function:
Utils.conclude_program file in 
close_out file

File: config/travis_install.sh [hunk 3]
```diff
@@ -16,6 +16,7 @@ esac
 echo "yes" | sudo add-apt-repository ppa:$ppa
 sudo apt-get update -qq
 sudo apt-get install -qq ocaml ocaml-native-compilers #camlp4-extra opam
+sudo apt-get install -qq python-matplotlib
```
Comment (akalicki) [inline config/travis_install.sh]: can we add a pip require file and then have travis install that file instead?

File: compiler/lib/core.py [hunk 4]
```diff
@@ -20,15 +21,29 @@
...
-    """Call normal print() function, but return argument that was passed"""
+    """Plot distributions for long lists and call normal print() function,
+    but return argument that was passed"""
+    if type(args[0]) is list and len(args[0]) >= DIST_LENGTH:
+        print_dist(args[0])
+        return str(args[0])
     __builtins__.print(*args, **kwargs)
     return str(args[0])
 
+def print_dist(dist):
```
Comment (akalicki) [inline compiler/lib/core.py]: comment please!

**Pull request merged**